go/go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/oasislabs/oasis-core/go
 
 replace (
-	github.com/tendermint/tendermint => github.com/oasislabs/tendermint v0.33.4-oasis1
+	github.com/tendermint/tendermint => github.com/oasislabs/tendermint v0.33.4-ptrus // TODO: update tag.
```

Comment:
Do we also need a replace directive for websocket here (since there is one in the fork) or are they transitive?

Reply:
Yeah seems that indeed it is needed, will add it. I tested this by creating an invalid replace directive in our fork: https://github.com/oasislabs/tendermint/commit/ecece7f01b43f28cf6a004f5e5b2a0fce908b437  (which causes the build to break). However the oasis-core/go build did still succeed when depending on that version.

Which probably means that any project that would depend on `oasis-core/go` should also propagate the replaces we use in `go.mod` (otherwise the code would actually use the non replaced dependencies??)

Reply:
also this means we can avoid updating the dependencies in the fork, which is probably preferable as it would only cause pain on the next update of the tendermint fork